src/js/components/Central/Central.tsx

```diff
@@ -15,9 +15,17 @@ import { Profile } from '../../views/Profile';
 export type CentralProps = {
   activeChatId: number | null;
   headerStyles?: string;
+  newPeakReached: number;
+  setNewPeakReached: (state: number) => void;
 };
 
-const Central = ({ activeChatId, headerStyles }: CentralProps) => {
+const Central = ({
+  activeChatId,
+  headerStyles,
+  newPeakReached,
+  setNewPeakReached,
+}: CentralProps) => {
+  const [newPostAdded, setNewPostAdded] = useState<number>(0);
```

Comment:
Coś podobnego używają chyba w implementacji reduxa. Nie jest to jakieś super piękne rozwiązanie, ale jedynym lepszym które mi przychodzi do głowy byłyby konteksty albo dorzucenie do apki globalnego storu (reduxa). Łap approve